providers/direct/src/main/java/quarks/providers/direct/DirectProvider.java

```diff
@@ -63,8 +70,29 @@ public DirectTopology newTopology(String name) {
     public Future<Job> submit(Topology topology) {
         return submit(topology, new JsonObject());
     }
+    
+    /**
+     * {@inheritDoc}
+     * <p>
+     * The configuration JSON representation is summarized in the following table:
```

Comment:
I considered adding configuration properties to the javadocs of the Submitter.submit(), but because various implementations can support different configuration properties, it made more sense to document where it is currently supported.

will move to a new Configs interface